docs/single-process-images.md

```diff
@@ -0,0 +1,80 @@
+# Single-Process Images
+
+These images are currently used on [pulp operator](https://docs.pulpproject.org/pulp_operator/), but they can be used in docker-compose or podman-compose. You can find a compose example (not production-ready) [here](https://github.com/pulp/pulp-oci-images/tree/latest/images/compose).
```

Comment:
We should not say 'not production ready' anywhere.

Reply:
We're literally about to delete an encryption key: https://github.com/pulp/pulp-oci-images/issues/372

We need to think of an alternate solution for warning users if we do not propagate the "not production ready" phrase (which we already say elsewhere.)

Reply:
I agree that we need to clearly communicate changes to consumers of these container images. We need to regularly publish a change log in our docs. We could probably also communicate about changes in a thread on discourse.